m4/nm.m4

```diff
@@ -2,6 +2,6 @@ AC_DEFUN([J4STATUS_PLUGINS_PLUGIN_NM], [
     libnm_min_version=0.9
```

Comment:
If you don’t handle older versions, you must make sure this is updated properly. ;-)